docs/cloud/pouta/persistent-volumes.md

```diff
@@ -134,6 +157,14 @@ the virtual machine to avoid data loss!**
 openstack server remove volume <server> <volume>
 ```
 
+If you want to delete a volume and the data contained on it you can do. The data will be deleted for ever, it will not be recoverable.
```

Comment:
What about creating an own subsection for the deletion? Something like `## Deleting volumes`. Mostly for two reasons:
- If you have a subsection I think you can then use it as an anchor for linking to it. Instead of writing "see below" we could then write "see here", with "here" pointing directing to the subsection
- deleting a volume does not seem to belong nicely to a subsection whose name is "Creating and Attaching volumes"

Reply:
I will change this, but ...

In principle, I consider that out of the scope of this PR. In my opinion this article's structure is strange and confusing. What does [Subsequent additions](https://docs.csc.fi/cloud/pouta/persistent-volumes/#subsequent-additions) mean? why "detaching" is the last step for "Creating and attaching volumes with command line tools", why expanding is only explained for the web interface?

If you agree, I will create a new PR that fixes this article's weirdness. So we separate the two changes.

Reply:
Sounds good